### ISSUE TITLE
Loading a demo from Space is failing

### ISSUE THREAD

**Issue by osanseviero** [gradio-app/gradio#875]
### Describe the bug

This is happening for all demos

See 

```python
import gradio as gr

gr.Interface.load("spaces/eugenesiow/remove-bg").launch(debug=True)
```

https://huggingface.co/spaces/eugenesiow/remove-bg works but the code above in a colab does not

### Reproduction

```
import gradio as gr

gr.Interface.load("spaces/eugenesiow/remove-bg").launch(debug=True)
```

I get this error

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/gradio/routes.py", line 265, in predict
    output = await run_in_threadpool(app.launchable.process_api, body, username)
  File "/usr/local/lib/python3.7/dist-packages/starlette/concurrency.py", line 39, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/usr/local/lib/python3.7/dist-packages/anyio/to_thread.py", line 29, in run_sync
    limiter=limiter)
  File "/usr/local/lib/python3.7/dist-packages/anyio/_backends/_asyncio.py", line 818, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.7/dist-packages/anyio/_backends/_asyncio.py", line 754, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.7/dist-packages/gradio/interface.py", line 573, in process_api
    prediction, durations = self.process(raw_input)
  File "/usr/local/lib/python3.7/dist-packages/gradio/interface.py", line 616, in process
    processed_input, return_duration=True
  File "/usr/local/lib/python3.7/dist-packages/gradio/interface.py", line 531, in run_prediction
    prediction = predict_fn(*processed_input)
  File "/usr/local/lib/python3.7/dist-packages/gradio/external.py", line 322, in fn
    result = json.loads(response.content.decode("utf-8"))
  File "/usr/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

### System Info

```shell
Colab, gradio 2.8.13
```

### Severity

critical

**Comment by abidlabs** [comment]
Thanks @osanseviero, aware of this issue! It's fixed in `main`, but will release a new version to pypi to take care of this.

**Comment by julien-c** [comment]
also just shipped a change server-side that may alleviate that in the meantime too

**Comment by abidlabs** [comment]
Fixed now in `gradio==2.8.14`. And @julien-c's change should fix it for interfaces in the wild
